__tests__/commands/install.js

```diff
@@ -460,7 +460,8 @@ test.concurrent('install should resolve circular dependencies 2', (): Promise<vo
 });
 
 
-test.concurrent('install should respect NODE_ENV=production', (): Promise<void> => {
+// don't run this test in `concurrent`, it will affect other tests
+test('install should respect NODE_ENV=production', (): Promise<void> => {
   const env = process.env.NODE_ENV;
   process.env.NODE_ENV = 'production';
```

Comment:
(eg. something like a config module that wraps it and could allow us to do `config.setEnvironment('production')`)